src/components/AlertBox/AlertBox.unit.spec.jsx

```diff
@@ -119,4 +119,47 @@ describe('<AlertBox />', () => {
         backgroundOnly
       />,
     ));
+
+  describe('analytics event', function () {
+    it('should be triggered when link in AlertBox is clicked', () => {
+      const handleAnalyticsEvent = e => {
+        expect(e.detail).to.eql({
+          componentName: 'AlertBox',
+          action: 'linkClick',
+          details: {
+            clickLabel: 'with a link',
+            headline: 'Headline',
+            status: 'info',
+            backgroundOnly: true,
+            closeable: false,
+          },
+        });
+      };
+
+      global.document.body.addEventListener(
+        'component-library-analytics',
+        handleAnalyticsEvent,
+      );
+
+      const wrapper = mount(
+        <AlertBox headline={Headline} status="info" backgroundOnly>
+          <div>
+            This is content
+            <a href="https://www.va.gov/" id="test-link-1">
+              with a link
+            </a>
+            .
+          </div>
+        </AlertBox>,
+      );
+
+      const testLink = wrapper.find('a');
+      testLink.simulate('click');
+
+      global.document.body.removeEventListener(
+        'component-library-analytics',
+        handleAnalyticsEvent,
+      );
+    });
```

Comment:
Since the `expect` checking is done in the event handler, it isn't really obvious what we expect if we are just reading this portion of the test. Because of that, can you try to rewrite it so that it resembles:

```js
      const testLink = wrapper.find('a');
      testLink.simulate('click');
      
      expect(eventHandler.calledWith(eventArgs)).to.be.true)
```

This is assuming that `eventHandler` was set up previously with something like `sinon.spy()`.

Reply:
Thanks for taking care of the eslint stuff this AM.

Let me know if this is what you were thinking as far as testing:

https://github.com/department-of-veterans-affairs/component-library/blob/8258a488b1bc3212bc0bed009396aea0c0382541/src/components/AlertBox/AlertBox.unit.spec.jsx